conda/recipes/strings_udf/meta.yaml

```
@@ -17,10 +17,22 @@ build:
   number: {{ GIT_DESCRIBE_NUMBER }}
   string: cuda_{{ cuda_major }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
   script_env:
-    - VERSION_SUFFIX
     - PARALLEL_LEVEL
-  # libcudf's run_exports pinning is looser than we would like
+    - VERSION_SUFFIX
+    - CMAKE_GENERATOR
+    - CMAKE_C_COMPILER_LAUNCHER
+    - CMAKE_CXX_COMPILER_LAUNCHER
+    - CMAKE_CUDA_COMPILER_LAUNCHER
+    # TODO: remove "gha" suffix before PR is merged
+    - SCCACHE_S3_KEY_PREFIX=libcudf-aarch64-gha # [aarch64]
+    - SCCACHE_S3_KEY_PREFIX=libcudf-linux64-gha # [linux64]
```

Leave this suggestion open until we're ready to merge, then apply it.
```suggestion
    - SCCACHE_S3_KEY_PREFIX=libcudf-aarch64 # [aarch64]
    - SCCACHE_S3_KEY_PREFIX=libcudf-linux64 # [linux64]
```

---

resolved in 85b1ee1